laundry-management-service/api/build.gradle

```diff
@@ -0,0 +1,39 @@
+plugins {
+    id 'java-library'
+    id 'com.google.protobuf' version '0.8.8'
+    id 'maven-publish'
+}
+
+repositories {
+    mavenCentral()
+}
+
+group = 'com.microservices.api'
+version = '0.0.1'
+
+dependencies {
+    implementation 'com.google.protobuf:protobuf-java:3.8.0'
+}
+
+protobuf {
+    generatedFilesBaseDir = "$projectDir/src"
+}
+
+sourceSets {
+    main {
+        proto {
+            srcDir "$projectDir/src/main/java/com/microservices/laundrymanagementapi/shema"
+            include '**/*.proto'
+        }
+    }
+}
+
+publishing {
+    publications {
+        myLibrary(MavenPublication) {
+            from components.java
+        }
+    }
+}
+
+testClasses.finalizedBy publishToMavenLocal
```

Comment:
why it is needed

Reply:
so that publishToMavenLocal is executed when building the project, I tied it to the last task (its testClasses) that is being performed. By default this task (publishToMavenLocal)  is not executed